tendrl/commons/flows/create_cluster/ceph_help.py

```diff
@@ -70,9 +70,9 @@ def install_packages(parameters):
     mon_ips = []
     osd_ips = []
     for node, config in parameters["Cluster.node_configuration"].iteritems():
-        if "mon" in config["role"].lower():
```

Comment:
these changes not required, create and import flows arent called often, And what if someone passes "mOn" or some value which we havent checked for